docs/scenes/geometry-browser.html

```diff
@@ -34,6 +34,7 @@
 				ConeGeometry,
 				Curve,
 				CylinderGeometry,
+				CapsuleGeometry,
```

Comment:
You can't import `CapsuleGeometry` from the core because it does not exists there.

If you are refering to `examples/jsm/math/Capsule.js`, this is not a geometry generator.